bfvmm/include/vmcs/vmcs_intel_x64.h

```diff
@@ -29,6 +29,13 @@
 #include <intrinsics/vmx_intel_x64.h>
 #include <intrinsics/msrs_intel_x64.h>
 
+extern void
```

Comment:
These should be right above the VMCS fields as a helper, written using generic programming defined int he header and not in the .cpp